source/geod24/concurrency.d

```diff
@@ -1608,22 +1754,25 @@ final public class Channel (T)
 
     ***********************************************************************/
 
-    private bool tryWrite () (auto ref T val) nothrow
+    private bool tryWrite () (auto ref T val, SelectState caller_sel = null, int caller_sel_id = 0) nothrow
```

Comment:
I see that some of the documentation for these parameters was actually introduced in the previous commit. This should be fixed.